docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -29,592 +34,661 @@ yb-tserver [ flags ]
 
-##### --follower_unavailable_considered_failed_sec
+Default: 0
+{{% /tags/wrap %}}
 
-The duration, in seconds, after which a follower is considered to be failed because the leader has not received a heartbeat. The follower is then evicted from the configuration and the data is re-replicated elsewhere.
+Enables [time travel queries](../../../manage/backup-restore/time-travel-query/) by specifying a Unix timestamp. After setting the parameter, all subsequent read queries are executed as of that read time, in the current session. Other YSQL sessions are not affected.
 
-Default: `900` (15 minutes)
+To reset the session to normal behavior (current time), set `yb_read_time` to 0.
 
-The `--follower_unavailable_considered_failed_sec` value should match the value for [--log_min_seconds_to_retain](#log-min-seconds-to-retain).
-
-##### --evict_failed_followers
-
-Failed followers will be evicted from the Raft group and the data will be re-replicated.
-
-Default: `true`
-
-##### --leader_failure_max_missed_heartbeat_periods
-
-The maximum heartbeat periods that the leader can fail to heartbeat in before the leader is considered to be failed. The total failure timeout, in milliseconds (ms), is [--raft_heartbeat_interval_ms](#raft-heartbeat-interval-ms) multiplied by `--leader_failure_max_missed_heartbeat_periods`.
-
-For read replica clusters, set the value to `10` in all yb-tserver and yb-master configurations.  Because the data is globally replicated, RPC latencies are higher. Use this flag to increase the failure detection interval in such a higher RPC latency deployment.
-
-Default: `6`
-
-##### --leader_lease_duration_ms
-
-The leader lease duration, in milliseconds. A leader keeps establishing a new lease or extending the existing one with every consensus update. A new server is not allowed to serve as a leader (that is, serve up-to-date read requests or acknowledge write requests) until a lease of this duration has definitely expired on the old leader's side, or the old leader has explicitly acknowledged the new leader's lease.
-
-This lease allows the leader to safely serve reads for the duration of its lease, even during a network partition. For more information, refer to [Leader leases](../../../architecture/transactions/single-row-transactions/#leader-leases-reading-the-latest-data-in-case-of-a-network-partition).
+Write DML queries (INSERT, UPDATE, DELETE) and DDL queries are not allowed in a session that has a read time in the past.
 
-Leader lease duration should be longer than the heartbeat interval, and less than the multiple of `--leader_failure_max_missed_heartbeat_periods` multiplied by `--raft_heartbeat_interval_ms`.
+##### --time_source
```

Comment:
This appears to be in the wrong spot, it is a flag not a parameter

Reply:
Okay, understood, I will fix it.